utils/orca_proxy/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx:1.18.0-alpine
+
+COPY nginx.conf /etc/nginx/nginx.conf
+
+CMD ["nginx"]
```

Comment:
It will be great to provide new line at the end of the file.

Reply:
Done